src/server/src/webhooks/pull_request.js

```diff
@@ -70,9 +78,19 @@ async function updateStatusAndComment(args, item) {
             let checkResult
             try {
                 checkResult = await cla.check(args, item)
-                logger.debug(`pullRequestWebhook-->updateStatusAndComment for the repo ${args.owner}/${args.repo}/pull/${args.number}`)
+                logger.debug({
+                    event: 'PR_WEBHOOK_DEBUG',
+                    msg: 'reporting status and comment on PR',
+                    repo: args.repo,
+                    owner: args.owner,
+                    number: args.number,
+                })
             } catch (error) {
-                logger.warn(new Error(error).stack)
+                logger.error({
```

Comment:
event is `WARNING`, but logger level is `logger.error` -- should these match (i.e. both warning or both error)? or keep as-is?

Reply:
good catch! 
changing `logger.error` to `logger.warn` to match previous logging level.